### DISCUSSION TITLE
Fussbodenheizung mit Homatic über ein HomeAssistant (Docker) auf dem Synology NAS steuern.

### DISCUSSION THREAD

**Discussion by Mannshoch** [OpenCCU/OpenCCU#2441] [🇩🇪 Fragen&Antworten]
Vor kurzem habe ich meine ersten Gehversuche mit Smarthome gemacht. Da ich die Anforderung an mich stelle, dass keine Klaut verwenden zu wollen wird schnell alles schwieriger. Der erste versuch mit einem Shelly Thermometer via MQTT war sehr gut gelungen. 

Nun solls wohl etwas komplexer werden. Ich möchte die Fussbodenheizungssteuerung in einem Raum Smart steuern und somit durch ein Homatic Gerät ersetzen.
Wir haben Danfoss 230V Ventile im Einsatz daher benötige ich wohl dieses Steuergerät:
`Smart Home Funk-Wandthermostat mit Schaltausgang 230 V für Markenschalter – HmIP-BWTH`

Bislang dachte ich, dass das nicht sinnvoll umsetzbar ist, weil ich sehr teuer noch ein Homatic CCU3 (laut Produktbeschreibung wird CCU2 nicht unterstützt) kaufen müsste, dann bin ich über dieses Projekt gestolpert. Wenn ich das umsetzen möchte, kann ich den Adapter `HmIP-RFUSB` verwenden. Leider werden im Wiki und anderswo viele technische Begriffe verwendet die ich nicht zuordnen kann. 

Meine Fragen:
* Kann mir wer bestätigen, dass ich das so korrekt ermittelt habe?
* Wenn das `HmIP-RFUSB` auf der 868Mhz Frequenz läuft, könnte ich das auch verwenden, um meine Somfy Markise zu steuern welche 868.96Mhz (laut Aufschrift des Bedienteils) nutzt?

**Comment by Baxxy13** [comment]
Du musst halt irgendwo eine Instanz von RaspberryMatic laufen lassen die Zugriff auf den HmIP-RFUSB hat.
Vielleicht per [Synology-VMM](https://github.com/jens-maus/RaspberryMatic/wiki/Installation-Synology-VMM).
Die RaspberryMatic oder besser gesagt die Homematic / Homematic-IP Geräte "integriert" man dann in den Homeassistant mittels [Homematic(IP) Local](https://github.com/jens-maus/RaspberryMatic/wiki/HomeAssistant-Integration).

Wenn am Einbauort für das HmIP-BWTH L+N sowie die "Steuerleitungen" zum Stellantrieb vorhanden sind dann sollte das soweit funktionieren.

Somfy kann nicht mit dem HmIP-RFUSB gesteuert werden. Zwar annähernd gleiche Frequenz aber definitiv andere "Sprache".

Planungsfragen bezüglich Homematic sind im Homematic-Forum allgemein besser aufgehoben.

**Reply by Mannshoch** [reply]
Wenn ich mir das [addon hier](https://www.home-assistant.io/blog/2022/02/07/improved-homematic-support/) anschaue, müsste es doch ohne zusätzliche VM gehen?

Ich habe mich bei homatic umgesehen, aber da ich ja kein teures ccu wollte bin ich am Schluss hier gelandet. Eine Weile lang bin ich in Foren am Suchen gewesen, wo man sich erst eine "CUL" Platine zusammenlöten musste, damit man nachher die Geräte koppeln konnte. Da reinzufinden ist schwer.

**Reply by Baxxy13** [reply]
Probier's aus. 
Vermutlich geht das nicht weil ja HA als Docker-Installation keine AddOn's unterstützt.
Dann müsstest du vielleicht HA als VM installieren und dann innerhalb der HA-VM RM als AddOn.
Bliebe die Frage ob das dann mit dem durchreichen des HmIP-RFUSB in die HA-VM zum RM-AddOn klappt.
